app/controllers/help_controller.rb

```diff
@@ -35,6 +35,10 @@ def signed_in
   def create
     @support_form = SupportForm.new(support_form_params)
 
+    if @support_form.email.blank?
```

Comment:
Would this affect any other place we've used a support form? e.g. offline? or is this the only place?

Reply:
This does evaluate to `true` if you post empty emails on any of the other forms, but the next line does nothing if `current_user` isn't present. 

The `&.` on that [next line](https://github.com/alphagov/govwifi-admin/pull/413/files/2ce3d029250e93de7d24cd535e2ef2504a411614#diff-32e4cccbb561b8d9e869a635fbcba1eeR39) is the "[safe navigator](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/)" operator. It's a terser way to write `unless thing_you_are_working_with.nil?`, but using it might sometimes make things less readable.

I also tried:

```
if @support_form.email.blank? & current_user.present?
  @support_form.email = current_user.email
end
```

but felt that long test was less readable.